app/javascript/components/pages/AboutUs.js

```diff
@@ -0,0 +1,8 @@
+import React, { Component } from "react";
+
+class AboutUs extends Component {
+  render() {
+    return <div></div>;
```

Comment:
👀

Reply:
How is the AboutUs component being verified as rendering on App?

Reply:
We just started that when we were waiting for an Adult to help us.

Reply:
👍